api/python/cellxgene_census/tests/conftest.py

```diff
@@ -49,7 +49,6 @@ def small_mem_context() -> soma.SOMATileDBContext:
     cfg = {
         "tiledb_config": {
             "soma.init_buffer_bytes": 32 * 1024**2,
-            "vfs.s3.no_sign_request": True,
         },
     }
     return soma.SOMATileDBContext().replace(**cfg)
```

Comment:
this pattern is obsolete - should be:

> soma.SOMATileDBContext(tiledb_config=cfg)

which avoids the create/delete/create waste